markdown/developers/tutorials/dsl2_modules_tutorial.md

```diff
@@ -11,414 +11,298 @@ If you create a new module with the goal of contributing the code to *nf-core*,
 
 ### Module guidelines
 
-The nf-core community has agreed on a minimal set of [guidelines](https://github.com/nf-core/modules#guidelines), intended to make module most suitable for general use, i.e. to be shared across a wide variety of community workflows.
+The nf-core community has agreed on a minimal set of [guidelines](https://nf-co.re/developers/modules#guidelines), intended to make module most suitable for general use, i.e. to be shared across a wide variety of community workflows.
```

Comment:
I cannot add a comment where there isn't a change but the paragraph before introduction says the tutorial uses as use-case the FGBIO module (previous tutorial), this one seems to be using FASTQC